src/plugins/getConfig.ts

```diff
@@ -17,6 +17,9 @@ export async function getConfig (): Promise<EnvConfigI> {
   const businessesUrl = process.env.VUE_APP_BUSINESSES_URL
   sessionStorage.setItem('BUSINESSES_URL', businessesUrl)
 
+  const corporateOnlineUrl = process.env.VUE_APP_CORPORATE_ONLINE_URL
+  sessionStorage.setItem('CORPORATE_ONLINE_URL', corporateOnlineUrl)
```

Comment:
I know some UIs don't bother putting the env keys in session storage. I did this to stay consistent here.

Reply:
Sev, I merged for the time being. I'll be working on my ticket and along the way I'll investigate if this can be simplified and the way I'll be using the Colin link. Is that OK with you?

Reply:
\* Perfect \*